tests/integration/corrections.rs

```diff
@@ -46,5 +76,95 @@ async fn test_corrections_by_day_auth() {
         .await
         .expect("Failed to GET");
     assert_eq!(r.status(), 200);
-    assert_eq!(r.text().await.unwrap(), "20220328");
+}
+
+async fn get_authed_path(path: &str, password: &str) -> Response {
+    let client = reqwest::Client::new();
+    let r = client
+        .get(path)
+        .basic_auth("user", Some(password))
+        .send()
+        .await
+        .expect("Failed to GET");
+    r
+}
+
+#[tokio::test]
+async fn test_corrections_by_day_result() {
+    let app = spawn_app().await;
+    let refunds = RefundModel {
+        db_pool: &app.db_connection(),
+    };
+    let subs = SubscriptionModel {
+        db_pool: &app.db_connection(),
+    };
+    make_test_refunds(&refunds, &subs).await;
+    let expected_refund = refunds.fetch_one_by_refund_id("refund_2").await.unwrap();
+    let expected_sub = subs
+        .fetch_one_by_subscription_id(&expected_refund.subscription_id)
+        .await
+        .unwrap();
+
+    // Path with no expected refund
+    let path = app.build_url("/corrections/2020-01-01.csv");
+    let r = get_authed_path(&path, &app.settings.authentication).await;
+    assert_eq!(r.status(), 200);
+    let actual_body = r.text().await.unwrap();
+    let expected_body = format!(
+        r#"
+&CID={}
+&SUBID={}"#,
+        app.settings.cj_cid, app.settings.cj_subid
+    );
+    assert_eq!(actual_body.trim(), expected_body.trim());
+
+    // Path with expected refund
+    let path = app.build_url("/corrections/2021-11-07.csv");
+    let r = get_authed_path(&path, &app.settings.authentication).await;
+    assert_eq!(r.status(), 200);
+    let actual_body = r.text().await.unwrap();
+    let expected_body = format!(
+        r#"
+&CID={}
+&SUBID={}
+RETRN,,{}"#,
+        app.settings.cj_cid, app.settings.cj_subid, expected_sub.id
+    );
+    assert_eq!(actual_body.trim(), expected_body.trim());
+}
+
+#[tokio::test]
+async fn test_corrections_today() {
+    let app = spawn_app().await;
+    let refunds = RefundModel {
+        db_pool: &app.db_connection(),
+    };
+    let subs = SubscriptionModel {
+        db_pool: &app.db_connection(),
+    };
+    make_test_refunds(&refunds, &subs).await;
+    let expected_refund_1 = refunds.fetch_one_by_refund_id("refund_1").await.unwrap();
+    let expected_refund_2 = refunds.fetch_one_by_refund_id("refund_3").await.unwrap();
```

Comment:
Just want to be absolutely clear that `expected_refund_2` is supposed to fetch `refund_3` and this isn't a typo.

Reply:
yes, this is correct. it's the refund for "today"

Reply:
i can rename to "today_refund_1" and "today_refund_2" because those are the two refunds from today that we're trying to get back.